packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/endpoint_client/endpoint_client.dart

```diff
@@ -16,25 +21,78 @@ import 'package:built_collection/built_collection.dart';
 /// {@endtemplate}
 class EndpointClient {
   /// {@macro amplify_analytics_pinpoint_dart.endpoint_client}
-  EndpointClient(
-    this._appId,
-    this._fixedEndpointId,
-    this._pinpointClient,
-    this._globalFieldsManager,
-    this._endpointBuilder,
-  );
-
-  final String _appId;
-  final PinpointClient _pinpointClient;
+  @visibleForTesting
```

Comment:
Is this used in a test? I don't see it used.

On a related note, there don't seem to be many unit tests in analytics. Is there a plan to add more?

Reply:
I will make unit tests for the followup PR to refactor Endpoint Info Management